Source/Core/Core/PowerPC/Interpreter/Interpreter_FloatingPoint.cpp

```diff
@@ -22,9 +22,9 @@ enum class RoundingMode
   TowardsNegativeInfinity = 0b11
 };
 
-static void SetFI(UReg_FPSCR* fpscr, int FI)
+void SetFI(UReg_FPSCR* fpscr, u32 FI)
```

Comment:
Shouldn't this be static? I can't find it in any header file.

Reply:
It's within an anonymous namespace, so it already has internal linkage. ConvertToInteger below doesn't have `static` either.